app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -125,15 +125,11 @@ def response_authn_context
     if saml_request.requested_vtr_authn_contexts.present?
       resolved_authn_context_result.expanded_component_values
     else
-      saml_request.requested_aal_authn_context ||
+      FederatedProtocols::Saml.new(saml_request).aal ||
```

Comment:
[question] should the default be moved into the `aal` method? is there a time where we'd want that to return `nil`?

Reply:
See answer below for with comment in `saml_idp_controller.rb`